src/data/contributeData.tsx

```diff
@@ -0,0 +1,284 @@
+import { NodeData } from '../components/node/Node';
+import React from 'react';
+import Link from '@docusaurus/Link';
+
+export const contributeData: NodeData = {
+  links: [
```

Comment:
I'm sorry, but I really don't think a huge `Nodedata` object is more maintainable and easier to work with than JSX. This will cause extra pain if we ever want to change this page.

Reply:
I disagree to be honest, it means if we want to change any of the stucture of a "node" we can do so very easily in the Node.tsx file, I also dont see how this is harder to edit than the previous one.

Doing it this way also means that we can unit test this alot easier than before.